charts/pega/templates/pega-environment-config.yaml

```diff
@@ -64,7 +64,6 @@ data:
   PEGA_STREAM_URL: {{ .Values.stream.url }}
 {{- end }}
 {{- end }}
-{{ if .Values.hazelcast.enabled }}
   HZ_DISCOVERY_K8S: "true"
   HZ_CLIENT_MODE: "true"
 {{ if .Values.hazelcast.client.clusterName }}
```

Comment:
Does this mean we mandate hazelcast c/s mode for all Pega platform versions? I'm not sure of the older versions 8.3.x or 8.4.x behavior on how these flags behave while starting up the web/batch pods.

Reply:
We are now restricting to use only C/S for both CMC and Pega Cloud and this will be applicable for versions from 8.6 as we are supporting CloudK from 8.6 only.

Reply:
I am not sure if these changes are backward compatible.  
Let's consider an example in CMC.
1) If I use this latest version of helm charts with older version of docker image (>8.6 pega docker image)where client server hazelcast mode is not supported, how these changes are expected to behave.